data-pipeline/assessment-aggregator/src/main/config/assessment-aggregator.properties

```diff
@@ -69,6 +69,8 @@ job.coordinator.replication.factor=1
 
 output.failed.topic.name=__env__.telemetry.assess.failed
 output.metrics.topic.name=__env__.pipeline_metrics
+output.prometheus.metrics.topic.name=__env__.telemetry.metrics
```

Comment:
Can we rename the topic as pipeline_metrics.v2 so that we have continuity in the naming convention

Reply:
The data products have been pushing the standardized metrics to __env__.telemetry.metrics topic. Since the data products metrics are going to be consumed by prometheus, I assumed the data pipeline will also emit these metrics to the same topic.

Reply:
ok